rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -0,0 +1,80 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_ansible
+
+%global foreman_dir /usr/share/foreman
+%global foreman_bundlerd_dir %{foreman_dir}/bundler.d
+
+Summary: Ansible integration with Foreman (theforeman.org)
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.2.1
+Release: 1%{?foremandist}%{?dist}
+Group:   Applications/System
+License: GPLv3
+URL:     https://github.com/theforeman/foreman_ansible
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: foreman >= 1.6.0
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}rubygems-devel
+Requires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: foreman-plugin >= 1.6.0
+
+BuildArch: noarch
+
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+Provides: foreman-plugin-ansible
+
+%description
+Ansible integration with Foreman.
+
+
+%package doc
+Summary: Documentation for %{name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+%setup -n %{pkg_name}-%{version} -q -c -T
+mkdir -p .%{gem_dir}
+%{?scl:scl enable %{scl} "}
+gem install --local --install-dir .%{gem_dir} \
+            --force %{SOURCE0} --no-rdoc --no-ri
+%{?scl:"}
+
+%build
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* %{buildroot}%{gem_dir}/
+
+%{foreman_bundlerd_file}
+
+%files
+%dir %{gem_instdir}
+%doc %{gem_instdir}/LICENSE
+%{gem_instdir}/app
+%{gem_libdir}
+%{gem_instdir}/locale
+%exclude %{gem_cache}
+%{gem_spec}
+%{foreman_bundlerd_dir}/%{gem_name}.rb
+%exclude %{gem_instdir}/test
+
+%files doc
+%doc %{gem_docdir}
```

Comment:
Interesting, does this macro work on el6? Tito tasks passed on f21 and el7 but el6 failed with `/var/tmp/rpm-tmp.EdOGhv: line 39: gem: command not found`